sources/engine/Stride.BepuPhysics/Stride.BepuPhysics.Navigation/Definitions/RecastNavigationConfiguration.cs

```diff
@@ -9,6 +9,10 @@ public class RecastNavigationConfiguration : Configuration
 {
     [Display("Build Settings", Expand = ExpandRule.Never)]
     public BuildSettings BuildSettings { get; set; } = new();
+
+    [Display("Pathfinding Settings", Expand = ExpandRule.Never)]
+    public PathfindingSettings PathfindingSettings { get; set; } = new();
+
```

Comment:
Just thinking out loud here, would it make sense to set the pathfinding settings per pathfinding query ? I guess we can have both, just food for though, feel free to do your thing

Reply:
That could make a lot of sense especially for large world pathfinding where you might want to have different LOD paths.. Ill see what can be done!